code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,489 @@
+	colour = "light pink"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that stops simple creatures from crossing its boundary."
+
+/obj/item/slimecross/warping/adamantine
+	colour = "adamantine"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that crystallizes materials based on nearby ore nodes."
+
+/obj/item/slimecross/warping/rainbow
+	colour = "rainbow"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that opens a gate to a pocket dimension."
+
+///////////////////////////////////////////////
+//////////////////SLIME RUNES//////////////////
+///////////////////////////////////////////////
+
+/obj/effect/slimerune/grey
+	name = "grey rune"
+	desc = "The center has an outline of a slime extract in it."
+
+/obj/effect/slimerune/grey/examine(mob/user)
+	..()
+	var/obj/item/slimecross/warping/grey/ex = extract
+	to_chat(user, "<span class='notice'>It has absorbed [ex.num_absorbed] extract[(ex.num_absorbed != 1) ? "s":""].</span>")
+	to_chat(user, "<span class='notice'>At 8 extracts, it can be manually fed an extract to produce a slime of that color.</span>")
+
+/obj/effect/slimerune/grey/process()
+	var/obj/item/slimecross/warping/grey/ex = extract
+	if(ex.num_absorbed >= 8)
+		return
+	if(locate(/obj/item/slime_extract) in get_turf(loc))
+		var/eaten = 0
+		for(var/obj/item/slime_extract/x in get_turf(src))
+			if(ex.num_absorbed >= 8)
+				break
+			qdel(x)
+			ex.num_absorbed++
+			eaten += 1
+		visible_message("<span class='notice'>[src] glows, and the extract[eaten != 1 ? "s":""] on it fade[eaten != 1 ? "":"s"] away.</span>")
+		if(ex.num_absorbed >= 8)
+			visible_message("<span class='notice'>[src] has finished its bluespace calibration, and can now produce a slime.</span>")
+
+/obj/effect/slimerune/grey/attackby(obj/item/slime_extract/O, mob/user)
+	var/obj/item/slimecross/warping/grey/ex = extract
+	if(!istype(O))
+		return
+	if(ex.num_absorbed < 8)
+		to_chat(user, "<span class='warning'>[src] isn't calibrated with enough slime extracts to produce a slime.</span>")
+		return
+	ex.num_absorbed = 0
+	user.visible_message("<span class='warning'>[user] places a slime extract onto [src], and a slime grows from it!</span>", "<span class='notice'>You place [O] onto [src], and a slime grows from it!</span>")
+	var/mob/living/simple_animal/slime/S = new(loc)
+	S.set_colour(O.slimecolor)
+	qdel(O)
+	return
+
+/obj/effect/slimerune/orange
+	name = "orange rune"
+	desc = "It's hot to the touch..."
+	var/obj/structure/bonfire/prelit/slime/fire
+
+/obj/effect/slimerune/orange/on_place()
+	fire = new(loc)
+
+/obj/effect/slimerune/orange/on_remove()
+	qdel(fire)
+
+/obj/structure/bonfire/prelit/slime
+	name = "magical flame"
+	desc = "It floats above the rune with constrained purpose."
+
+/obj/structure/bonfire/prelit/slime/CheckOxygen()
+	return TRUE
+
+/obj/structure/bonfire/prelit/slime/extinguish()
+	return
+
+/obj/effect/slimerune/purple
```

Comment:
Im not really particularly keen on this ones effects

Reply:
I'm definitely not too attached to it. Any ideas for health-related effects?